### PR TITLE
Corrected handling of ploidy in MergeGVCFUtils.createSNP()

### PR DESCRIPTION
**Pull request by tcasstevens**
Fixing Indels aren't exporting ploidy correctly

### CODE REVIEW

File: src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt [hunk 1]
```diff
@@ -167,20 +167,46 @@ private fun createSNP(
 
                     when {
 
-                        variant.isINS -> {
-                            symbolicAlleles.add("<INS>")
-                            Pair(variant.isPhased(0), listOf("<INS>"))
-                        }
+                        variant.isIndel -> {
```
Comment (zrm22) [inline src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt:170]: Do you have a unit test to verify that this change works correctly?

**Review by zrm22 — APPROVED** [review]

File: src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt (hunk 1 above)
Comment (tcasstevens) [reply src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt:170]: Working on it. Also, I might have committed these changes to master. Can you tell if I did that?

Comment (zrm22) [reply src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt:170]: Yeah it looks like there was 3 commits to master.  It's odd though that you still have a diff here then if it matches.

Comment (tcasstevens) [reply src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt:170]: Do you have suggestion how to proceed?

**Review by lynnjo — APPROVED** [review]

**Pull request merged**